js_modules/dagster-ui/packages/ui-core/src/workspace/WorkspaceRoot.tsx

```diff
@@ -45,12 +47,6 @@ const RepoRouteContainer = () => {
     );
   }
 
-  const {loading} = workspaceState;
-
-  if (loading) {
-    return <div />;
-  }
```

Comment:
does this change effect other pages that we should be concerned about?

Reply:
I went through them all and sensors was the only one that had an issue which I fixed